usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -53,79 +62,92 @@ def search_transactions(filters, fields, sort, order, lower_limit, limit):
         'query': {
             'query_string': {
                 'query': keyword
-                }
-            },
+            }
+        },
         'sort': [{
             query_sort: {
                 'order': order}
-            }]
```

Comment:
#todo change the look ups so we're sorting on `display_award_id`. Are we doing that with this PR or is it non-critical?

Reply:
I don't think that should occur until after we switch to the new indices. It would involve over-complicated code to handle both old and new documents

Reply:
That's why the PR description has "Started the code changes to use the new display_award_id if it exists in the documents"